gettsim/parameters/ges_renten_vers.yaml

```diff
@@ -639,3 +639,37 @@ zugangsfaktor_veränderung_pro_jahr:
   2001-01-01:
     vorzeitiger_renteneintritt: 0.036
     späterer_renteneintritt: 0.06
+
+regelaltersgrenze:
+  name:
+    de: >-
+      Piecewise linear welche die Regelaltersgrenze ausgibt. Bei Erreichen diesen Alters hat die Person einen Anspruch auf die Regelaltersrente.
```

Comment:
This is rather the `description`. Name should be much shorter.

Here is a description of the tags in the .yaml-files: https://github.com/iza-institute-of-labor-economics/gettsim/pull/148/files

(This is not consistently applied for all current parameters.. but better follow the guidelines directly for new parameters.)